### ISSUE TITLE
telegram-plane not working

### ISSUE THREAD

**Issue by Philipp97714** [FortAwesome/Font-Awesome#15033]
related to #13338

I downloaded a HTML template which has the tags
`<link href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.css" rel="stylesheet">`
and
`<i class="fa fa-telegram fa-2x" aria-hidden="true">`

however, if I change it to `<i class="fa fa-telegram-plane fa-2x" aria-hidden="true">` the icon disappears. I thought using the link above alway graps the latest version so it should be working. If I change fa to fab it isn't working either.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Same answer as #13338

[telegram-plane](https://fontawesome.com/icons/telegram-plane?style=brands) has been introduced in Font Awesome 5.0.0.

> https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.css
> I thought using the link above alway graps the latest version so it should be working

It is a link to the latest Font Awesome 4 version available. Please use our new CDN

Please take a look at our [Start page](https://fontawesome.com/start)

> `<i class="fa fa-telegram fa-2x" aria-hidden="true">`

Please also update the syntax. You need `<i class="fab fa-telegram-plane fa-2x" aria-hidden="true">` (note the extra `b`)

Working demo: https://jsfiddle.net/tagliala/ryncaLg7/3/

If you still think there is an issue with Font Awesome, fill out our [bug report template](https://github.com/FortAwesome/Font-Awesome/issues/new?template=bug-report.md)
